app/assets/javascripts/disqus.js

```diff
@@ -0,0 +1,14 @@
+loadDisqus = function(shortname, title, url, identifier) {
+/* * * DON'T EDIT BELOW THIS LINE * * */
+var disqus_shortname = shortname;
+var disqus_title = title;
+var disqus_url = url;
+var disqus_identifier = identifier;
+
+DISQUS=function(a){"use strict";var b=a.DISQUS||{};return b.define=function(c,d){"function"==typeof c&&(d=c,c="");for(var e=c.split("."),f=e.shift(),g=b,h=(d||function(){return{}}).call({overwrites:function(a){return a.__overwrites__=!0,a}},a);f;)g=g[f]?g[f]:g[f]={},f=e.shift();for(var i in h)h.hasOwnProperty(i)&&(!h.__overwrites__&&null!==g[i]&&g.hasOwnProperty(i)?b.logError&&b.logError("Unsafe attempt to redefine existing module: "+i):g[i]=h[i]);return g},b.use=function(a){return b.define(a)},b.define("next"),b}(window);
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.<br>Identifier '__overwrites__' is not in camel case.<br>Missing semicolon.<br>Don't make functions within a loop.<br>Expected '{' and instead saw 'g'.<br>Expected '{' and instead saw 'h'.<br>The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.